validator/client/src/main/java/tech/pegasys/teku/validator/client/loader/ValidatorSourceFactory.java

```diff
@@ -147,7 +147,9 @@ private boolean ensurePathExists(final Path directory) {
             config,
             externalSignerHttpClientFactory,
             publicKeyLoader,
-            asyncRunner);
+            asyncRunner,
+            true // TODO Check this
+            );
```

Comment:
I suspect this instance of `ExternalValidatorSource` will set read only to false since it's loading validators from the command line and then we'd create a second instance of `ExternalValidatorSource` with read only set to true in `addMutableValidatorSource` (so it would create both a `LocalValidatorSource` to store in `mutableLocalValidatorSource` and a `ExternalValidatorSource` which would be stored in a new `mutalbeExternalValidatorSource` field here and that would also be passed to `ValidatorLoader`).